.github/workflows/release-v2.yml

```diff
@@ -285,13 +373,54 @@ jobs:
       - name: Build and push (coordinator, Amazon ECR)
         if: ${{ !inputs.skipPublish }}
         run: |
-          ./build_docker_images.sh -p -t ${{needs.resolve-tag.outputs.release-tag}} -r ${{ secrets.ECR_REPOSITORY }} -a
+          ./build_docker_images.sh -p -t ${{needs.resolve-tag.outputs.release-tag}} -r ${{ secrets.ECR_REPOSITORY }}
```

Comment:
more changes ported from main branch

Reply:
the `-a` was remove in main?

Reply:
yes. `-a` is not one of the valid options for the `build_docker_images.sh` and would result in an error.